scripts/test-files/windows2k22-installer-pipelinerun.yaml

```diff
@@ -1,13 +1,19 @@
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  generateName: windows2k22-installer-run-
+  generateName: windows2k22-installer-
```

Comment:
Keep `-run` suffix like in other pipelineruns?

Reply:
fixed